agent/acs/handler/acs_handler.go

```diff
@@ -78,8 +79,8 @@ const (
 	acsProtocolVersion = 2
 	// numOfHandlersSendingAcks is the number of handlers that send acks back to ACS and that are not saved across
 	// sessions. We use this to send pending acks, before agent initiates a disconnect to ACS.
-	// they are: refreshCredentialsHandler, taskManifestHandler, and payloadHandler
-	numOfHandlersSendingAcks = 3
+	// they are: taskManifestHandler, and payloadHandler
+	numOfHandlersSendingAcks = 2
```

Comment:
Wow this variable's description sounds like a hack! 

Anyway, do you mind providing some more details in the PR description on why `refreshCredentialsHandler` is free from this logic now?

Reply:
😂 - good thing it's going away!

Sure, I've added some context on this to the PR description.